.goreleaser.yml

```diff
@@ -18,15 +18,8 @@ builds:
       - -X code.cloudfoundry.org/cli/version/binaryVersion={{.Version}}
     goos:
       - linux
-      - darwin
```

Comment:
no need to remove those, please keep them in here.

Reply:
@benjaminguttmann-avtq it's ok for me